diesel_tests/tests/insert.rs

```diff
@@ -452,6 +452,47 @@ fn insert_returning_count_returns_number_of_rows_inserted() {
     assert_eq!(1, second_count);
 }
 
+#[test]
+#[cfg(not(any(feature = "mysql", feature = "sqlite")))]
+fn insert_with_generated_column() {
+    use crate::schema::user_with_last_names::table as users;
+    #[derive(Debug, Queryable, Insertable, Selectable, Default)]
+    struct UserWithLastName {
+        id: i32,
+        first_name: String,
+        last_name: String,
+        #[diesel(skip_insertion)]
+        full_name: String,
+    }
+
+    let connection = &mut connection();
+    diesel::sql_query(
+        "CREATE TABLE user_with_last_names (
+        id SERIAL PRIMARY KEY,
+        first_name VARCHAR NOT NULL,
+        last_name VARCHAR NOT NULL,
+        full_NAME VARCHAR GENERATED ALWAYS AS (first_name || ' ' || last_name) STORED
+    )",
+    )
+    .execute(connection)
+    .unwrap();
+    let new_users: &[_] = &[UserWithLastName {
+        first_name: "Sean".to_string(),
+        last_name: "Black".to_string(),
+        ..Default::default()
```

Comment:
Looks like a usage of this feature would always end up containing a part that's going to look like this where you have dummy values...
What happens with the ID though? It doesn't have the attribute so it looks like you are somewhat-implicitly setting it to zero?
And that looks encouraged by this feature...

Reply:
I wasn't intending to show anything related to the id/primary key field here, so I'll remove that from the test as it's not useful. I really just want to be able to insert into tables with generated columns without having to convert into some intermediate type first.